erpnext/stock/doctype/serial_no/serial_no.py

```diff
@@ -431,7 +431,6 @@ def update_args_for_serial_no(serial_no_doc, serial_no, args, is_new=False):
 		"via_stock_ledger": args.get("via_stock_ledger") or True,
 		"supplier": args.get("supplier"),
 		"location": args.get("location"),
-		"sales_order": frappe.db.get_value(args.get("voucher_type"), args.get("voucher_no"), "sales_order_no"),
```

Comment:
@imdadhussain remove this file change, track it from https://github.com/Bloomstack/erpnext/pull/1742

Reply:
@sahil28297 it is merge on develop so that we can pull it on local.